examples/azure/postgres/build.sh

```diff
@@ -31,12 +32,28 @@ server <= azure_postgres_set_max_size($server, $size)
 server <= azure_postgres_set_version($server, $version)
 
 if $ssl == "false" {
-        server <= azure_postgres_disable_ssl($server)
+	server <= azure_postgres_disable_ssl($server)
 }
 
 azure_postgres_server_create($server)
 
 echo "created Postgres Server"
+echo "creating Postgres Server Firewall rule"
+
+publicip, status <= curl ipinfo.io/ip
```

Comment:
This was just a stupid idea I would like to run by you guys and confirm it's stupid.